src/utils/api-utils.ts

```diff
@@ -0,0 +1,62 @@
+import { BackendClient } from "../clients/api";
+import { ICustomWindow } from "./customTypes/CustomWindow";
+
+/* function to define a Backend instance with base url set */
+// tslint:disable-next-line:no-any
+export const baseUrlBackendClient = (token: string): any => {
+  const customWindow = (window as unknown) as ICustomWindow;
+
+  const url =
+    customWindow._env_.IO_ONBOARDING_PA_API_HOST +
+    ":" +
+    customWindow._env_.IO_ONBOARDING_PA_API_PORT;
+
+  return BackendClient(url, token);
+};
+
+export interface IErrorWithStatus extends Error {
+  status: string;
+}
+
+/* function to manage service responses */
+export const manageApiResponse = (
+  response: Partial<ReturnType<typeof BackendClient>>
```

Comment:
I don't find any value added by this method. it's better to handle the success response (or error) "inline" (where it occurs). The only feature of this method here is to hide the error message which is bad : )